src/library/Uno.Themes.WinUI.Markup/Theme/ToggleButton.cs

```diff
@@ -0,0 +1,334 @@
+						public static ResourceValue<Brush> SelectedOverlay => new("IconToggleButtonBorderBrushSelectedOverlay", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "IconToggleButtonBorderBrushChecked")]
+						public static ResourceValue<Brush> Checked => new("IconToggleButtonBorderBrushChecked", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "IconToggleButtonBorderBrushCheckedPointerOver")]
+						public static ResourceValue<Brush> CheckedPointerOver => new("IconToggleButtonBorderBrushCheckedPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "IconToggleButtonBorderBrushCheckedPressed")]
+						public static ResourceValue<Brush> CheckedPressed => new("IconToggleButtonBorderBrushCheckedPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "IconToggleButtonBorderBrushCheckedDisabled")]
+						public static ResourceValue<Brush> CheckedDisabled => new("IconToggleButtonBorderBrushCheckedDisabled", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "IconToggleButtonBorderBrushIndeterminate")]
+						public static ResourceValue<Brush> Indeterminate => new("IconToggleButtonBorderBrushIndeterminate", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "IconToggleButtonBorderBrushIndeterminatePointerOver")]
+						public static ResourceValue<Brush> IndeterminatePointerOver => new("IconToggleButtonBorderBrushIndeterminatePointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "IconToggleButtonBorderBrushIndeterminatePressed")]
+						public static ResourceValue<Brush> IndeterminatePressed => new("IconToggleButtonBorderBrushIndeterminatePressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "IconToggleButtonBorderBrushIndeterminateDisabled")]
+						public static ResourceValue<Brush> IndeterminateDisabled => new("IconToggleButtonBorderBrushIndeterminateDisabled", true);
+
+						[ResourceKeyDefinition(typeof(Thickness), "IconToggleButtonBorderThickness")]
+						public static ResourceValue<Thickness> BorderThickness => new("IconToggleButtonBorderThickness", true);
+					}
+
+					public static class StateCircle
+					{
+						public static class Fill
+						{
+							[ResourceKeyDefinition(typeof(Brush), "IconToggleButtonStateCircleFill")]
+							public static ResourceValue<Brush> Default => new("IconToggleButtonStateCircleFill", true);
+						}
+
+						[ResourceKeyDefinition(typeof(double), "IconToggleButtonStateCircleOpacityPointerOver")]
+						public static ResourceValue<double> OpacityPointerOver => new("IconToggleButtonStateCircleOpacityPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(double), "IconToggleButtonStateCircleOpacityPressed")]
+						public static ResourceValue<double> OpacityPressed => new("IconToggleButtonStateCircleOpacityPressed", true);
+
+						[ResourceKeyDefinition(typeof(double), "IconToggleButtonStateCircleOpacityFocused")]
+						public static ResourceValue<double> OpacityFocused => new("IconToggleButtonStateCircleOpacityFocused", true);
```

Comment:
Should be placed inside of an Opacity class, with the properties being PointerOver, Pressed, and Focused